CMakeLists.txt

```diff
@@ -205,6 +205,50 @@ if(${IREE_ENABLE_CCACHE})
   endif()
 endif()
 
+
+#-------------------------------------------------------------------------------
+# IREE assertions
+# We don't love the way this is done, but we have to line it up with how LLVM
+# does it and not diverge, since all implementations and all header users must
+# have the same definition of NDEBUG.
+#
+# LLVM defaults this to ON for Debug builds only. If set ON manually, then
```

Comment:
Can you clarify what "this" is here? There is useful information in this comment but I can't quite follow it reading linearly.